games-strategy/freeciv/additional-files/freeciv.rdef.in

```diff
@@ -0,0 +1,32 @@
+// Freeciv icon by Stephanie Fu
+
+resource app_signature "application/x-vnd.Freeciv";
+
+resource app_flags B_EXCLUSIVE_LAUNCH;
+
+resource app_version {
+	major = @MAJOR@,
+	middle = @MIDDLE@,
+	minor = @MINOR@,
+	variety = B_APPV_DEVELOPMENT,
+	internal = @INTERNAL@,
+	short_info = "Freeciv",
+	long_info = "Freeciv, a civilization-like game.\nCopyright 2002-2015 The Freeciv Team"
+};
+
+resource file_types message;
+
+resource vector_icon array {
+	$"6E6369660C0200040200FFFFFFF6CA9F3602000402008B9CCDC51D41AB020004"
+	$"020082C97DBF0A72030200040200FFFFFFCE59820303BC9D5103085A01030643"
+	$"00030A257603010000038B480003FF0000010000015C080A04244936493D3A2B"
+	$"3A0A04244C24513651364C0A044F3D4F434852484C0A062451244C322E562E56"
+	$"3449510A043C3D3F3D35253225080F3429382A3E294028422745274829442A42"
+	$"2B412C3F2F3E303C323A33383406138AAA8A8E0030283B413E39343C333F3041"
+	$"2E422C442B452A47482947284427C1E627C0EB27413B29383634080549575455"
+	$"5E405B3D543E0D0A0B0107000A080103301F2401178400040A0301002026180A"
+	$"0201002031270A0101002038180A000100201F270A040101201F240A05010120"
+	$"31240A060102201F240A0701022026150A0801061001178300040A090104201F"
+	$"240A0A010500"
+};
+
```

Comment:
rdef file isn't updated, it still shows the first icon you created Stephanie, the iom file is correct